### PR TITLE
Windows: support ppid

### PR DESCRIPTION
**Pull request by Rummskartoffel**
Rename windows_restart.nim to windows_helpers.nim because that file does
a lot more than just :restart at this point.

### CODE REVIEW

File: src/native_main.nim [hunk 1]
```diff
@@ -308,8 +308,7 @@ proc handleMessage(msg: MessageRecv): MessageResp =
             when defined posix:
                 result.content = some($getppid())
             else:
```
Comment (bovine3dom) [inline src/native_main.nim:310]:
```suggestion
            elif defined windows:
```

File: src/native_main.nim [hunk 2]
```diff
@@ -308,8 +308,7 @@ proc handleMessage(msg: MessageRecv): MessageResp =
...
-                result.cmd = "error"
-                result.error = "ppid is not available on this OS"
+                result.content = some($getppidWindows())
```
Comment (bovine3dom) [inline src/native_main.nim:311]:
```suggestion
                result.content = some($getppidWindows())
            else: 
                result.cmd = "error"
                result.error = "ppid is not available on this OS"
```

**Review by bovine3dom — COMMENTED** [review]
Assuming all non-POSIX systems are Windows is probably safe but at the same time it feels dodgy to me

**Comment by Rummskartoffel** [comment]
It's almost definitely safe, since you're just going to get a compiler error if you have neither of those functions, but it can't hurt to prevent that also.

**Pull request merged**